addons/im_livechat/static/src/core/public_web/discuss_app_model_patch.js

```diff
@@ -3,6 +3,21 @@ import { fields } from "@mail/core/common/record";
 
 import { _t } from "@web/core/l10n/translation";
 import { patch } from "@web/core/utils/patch";
+import { debounce } from "@web/core/utils/timing";
+
+// Looking for help subscription is triggered when the sidebar category is
+// opened, and when the discuss app is active. To avoid unsubscribing right away
+// when the user closes the sidebar or switches to another app, wait for 5
+// minutes before unsubscribing.
+export const LFH_UNSUBSCRIBE_DELAY = 5 * 60 * 1000;
+const debouncedUnsubscribeLivechatHelp = debounce((discussApp) => {
+    discussApp.store.env.services.bus_service.deleteChannel("im_livechat.looking_for_help");
```

Comment:
It's probably not reachable? But this should only delete if livechatHelpSubscribed was true.